config/routes.js

```diff
@@ -1,19 +1,50 @@
 const axios = require('axios');
-
-const { authenticate } = require('../auth/authenticate');
+const userModel = require('../database/models/user');
+const { authenticate, generateToken } = require('../auth/authenticate');
+const bcrypt = require('bcryptjs');
+const jwt = require('jsonwebtoken');
 
 module.exports = server => {
   server.post('/api/register', register);
   server.post('/api/login', login);
   server.get('/api/jokes', authenticate, getJokes);
+  
 };
 
-function register(req, res) {
-  // implement user registration
+async function register(req, res) {
+	try {
+		
+		r = await userModel.addUser({username:req.body.username,password:req.body.password})
+		if(!r) throw "db error";
```

Comment:
could be simplified by just sending req.body as the argument on the addUser function